CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## [2.0.120230216]
+
+* CLI version update that address [issue](https://github.com/Azure/Bridge-To-Kubernetes/issues/162), [issue 89](https://github.com/Azure/Bridge-To-Kubernetes/issues/89) and [issue 56](https://github.com/Azure/Bridge-To-Kubernetes/issues/56)
```

Comment:
👍 Nice, looks good, please feel free to add information about:

* Added `redact` funct for logging information under [Fix/readact creds from logging](https://github.com/Azure/vscode-bridge-to-kubernetes/commit/73bc3d8218c47eacadadc23d59a5f442346ef28e)

Reply:
Didnt noticed this was already checked in, thanks!